### PR TITLE
Add party invite macro

### PR DESCRIPTION
**Pull request by Stallind**
Binding a macro to say /add doesn't work so I added a macro to invite by target

Update:
Added more commands 

https://gyazo.com/f1dc346dd2d95fba5f1212ed37805d72

### CODE REVIEW

**Comment by Stallind** [comment]
Figured why not just add them all -> Add, accept, quit =)

**Comment by Alias-Alan** [comment]
This hits on the problem I had with the info command. I had to do the same thing and had to make an actual macro to execute the command because for whatever reason say isn't recognizing symbols within the string.

File: src/Game/Managers/MacroManager.cs [hunk 1]
```diff
@@ -1783,6 +1831,10 @@ internal enum MacroSubType
         Right,
         Down,
         Left,
+        Add,
```
Comment (andreakarasho) [inline src/Game/Managers/MacroManager.cs]: These should be added at the end of the enum or you will break macroes that already exsit

**Comment by andreakarasho** [comment]
I want to be sure thst i ve added all common potions in the submacro enum before merging this one

**Pull request closed without merging**